spec.emu

```diff
@@ -99,7 +150,13 @@ contributors:
 </emu-clause>
 
 <emu-clause id="sec-modifications-of-weakset">
-  <h1>Modifications to the properties of WeakSet.prototype</h1>
+  <h1>Modifications to WeakSet</h1>
+
+  <emu-clause id="sec-weakset.isValidValue">
+    <h1>WeakMap.isValidValue( _value_ )</h1>
```

Comment:
Technically this predicate is answering if it's a valid *key* - everything's a valid value.